pheps/phep-9999.md

```diff
@@ -0,0 +1,162 @@
+
+*** = conditional upon justification that package is in danger of dropping down to a Silver tier 
+
+Descriptions for each heading are as follows:
+   - Summer School Inclusion: indicates whether a package will be included in summer school teaching materials
+   - PyHC-top-tier Env Inclusion: indicates whether a package will be included within the current PyHC software environment used at the summer school (also included within env in Science Platforms Coordination group???)
+   - PyHC-all Env Inclusion: indicates whether a package will be included within the current PyHC software environment containing all packages Bronze and higher. (Does this make sense based on the ability or not of a package to be installed in a common software env???)
+   - PyHC-Chat Bot Inclusion: indicates whether a packages will have up-to-date information included within the ChatGPT4-powered PyHC-Chat bot
+   - pyOpenSci Verified Badge: a badge that shows whether a package has completed the pyOpenSci review process
+   - Standards Compliance Assistance: indicates whether a package will receive extra help and/or funding (dependent on available PyHC resources) for conforming to the PyHC standards
+   - Main Page Listing: indictes whether a package's information will be displayed on a new, main PyHC Project page
+   - Secondary Page Listing: indicates whetheer a package's information will be displayed on a new, secondary PyHC Project page 
+   - HSSI Inclusion: indicates whether a package will be included within the Heliophysics Software Search Interface
+   - Conference Travel Funding: indicates whether developers from a package will be considered for travel funding assistance to relevant science conferences (e.g. SHINE, CEDAR, or GEM)
+
+Packages are evaluated against level of compliance with each requirement, as shown in the PyHC tiering chart. To be accepted for a tier, a package must meet **all** the requirements for said tier. Therefore, should a package fall into different tiers depending on the specific requirement, the package will be accepted at the lowest tier of requirements it meets.  
+
+For example, if a package meets some requirements for the Silver tier, but other requirements only meet the Bronze tier, the package will be considered a Bronze tier package.   
+
+
+# Package Tiering Implementation Process
+
+Following acceptance of this PHEP, the PyHC website will be updated to reflect the new tiers. The process is then as follows:
+- Within six months of PyHC website update (written communication provided in email form), packages must submit a PR to [the PyHC website GitHub](https://github.com/heliophysicsPy/heliophysicsPy.github.io) indicating what level they believe their package is
+    - need to include written justification on the PR for chosen tier 
+- Within three months of PR publish, PyHC Tech Lead reviews the package and asserts if self-evaluated level matches expectations
```

Comment:
possible we could have a small technical steering committee to help Shawn and I out with this? Thoughts?

Reply:
I'd appreciate any help since this basically means re-evaluating every PyHC package, but I could also imagine it being manageable myself if there's not much else going on and if self-evals trickle in over a 6-month period.

Reply:
Sounds useful. I can serve from the metadata / open science perspective.

Reply:
Awesome; I'll modify to include the leadership team + a technical steering committee. The latter is something I wanted to bring up in any case come the Fall meeting.